README.rst

```diff
@@ -2,10 +2,55 @@
 LSST the Docs: Mason
 ####################
 
-**Mason** is the document build tool for LSST software documentation that runs in the Jenkins build environment.
+**Mason** is the build tool for LSST software documentation that runs in the Jenkins build environment.
 It is a part of the **LSST the Docs** (LTD) service for continuous documentation deployment.
 You can learn more about LTD in our `SQR-006`_ technote.
 
+Installation
+============
+
+Mason is intended to work with either Python 2.7, 3.4, or 3.5.
+It *must* run from the same Python as the software being documented, however.
+
+::
+
```

Comment:
I suggest adding a `pyvenv`|`virtualenv` creation and activation example here only because DM seems to have many dev unfamiliar with these tools.